packages/models/test/fixtures/categories.mock.ts

```diff
@@ -60,12 +60,21 @@ const CATEGORIES_MAP = {
   },
 } satisfies Record<string, CategoryConfig>;
 
-type CategorySlug = keyof typeof CATEGORIES_MAP;
-
-export function categoryConfig(
-  slug: CategorySlug = 'performance',
-): CategoryConfig {
-  return CATEGORIES_MAP[slug];
+export function categoryConfig(slug = 'category-slug-1'): CategoryConfig {
```

Comment:
The old version had a realistic category prepared. Making it into a dummy config isn't a good change, I think.

Reply:
I would move this data into lighthouse package and provide a more realistic mock?

Reply:
I would keep it here, otherwise we're mixing `scope:plugin` with `scope:core`. And the categories are cross-plugin anyway. 

Why not just keep this function as it was?

Reply:
done